Filipe/judge.py

```diff
@@ -0,0 +1,29 @@
+# a ideia é que o programa conte cada letra de uma frase ou palavra
+
+sentenca = input ("Digite a sentença sem números assentos ou caracteres específicos: ")
+sentenca = sentenca.lower()
+
+letras = "abcdefghijklmnopqrstuvwxyz"
+contador = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
+
+numeros = [0,1,2,3,4,5,6,7,8,9]
+
+for j in range(len(sentenca)):
+    if sentenca[j] == " " or sentenca[j] in numeros:
+        continue
+    for i in range(len(letras)):
+        if sentenca[j] == letras[i]:
```

Comment:
Me parece que esse codigo pode ser bem simplificado se você usar um dicionario ao inves de 2 vetores. Um dicionario to tipo { letra : contador}. 

Você pode criar um bem facilmente com 
`dictionary={character : 0 for character in letras}`